src/Time/Tags.hpp

```diff
@@ -45,13 +45,21 @@ struct TimeStep : db::SimpleTag {
 
 /// \ingroup DataBoxTagsGroup
 /// \ingroup TimeGroup
-/// \brief Tag for compute item for current ::Time (from TimeId)
-struct Time : db::ComputeTag {
-  static std::string name() noexcept { return "Time"; }
+/// \brief Tag for compute item for ::Time of the current substep (from TimeId)
+struct SubstepTime : db::ComputeTag {
```

Comment:
Since you do this renaming, perhaps also rename the `Time` class to `SubstepTime` and `TimeId::time` to `TimeId::substep_time` to keep consistency?

Reply:
`Time` -> `SubstepTime`: no.  Same reason we don't rename `Variables` to `EvolvedVariables`: a description of the value is different from a description of the type.  (There's nothing inherently substep-related in `Time`.)

`TimeId::time` -> `TimeId::substep_time`: yes, good idea.